src/main.rs

```diff
@@ -152,6 +158,9 @@ struct SimulatorConfig {
     /// Show subscriber stats
     #[arg(long, default_value = "false")]
     show_sub_stat: bool,
+    /// Type of data to send
+    #[arg(long, default_value = "imu")]
+    data_type: String,
```

Comment:
can we have this to be an enum? will make it much more clearer when passing this argument 
(https://docs.rs/clap/latest/clap/trait.ValueEnum.html)

Reply:
also can we update the inconsistency with this argument (mentioned default and actual default are different)
<img width="470" alt="image" src="https://user-images.githubusercontent.com/13679906/217527593-10358a66-6933-438f-8123-8a1074f2f7c8.png">